utils/manifest-utils/src/keystore/golem_keystore.rs

```diff
@@ -58,14 +59,13 @@ impl KeystoreBuilder<GolemKeystore> for GolemKeystoreBuilder {
 }
 
 // Return certificate with its id
-fn read_cert(cert_path: &PathBuf) -> anyhow::Result<(String, GolemCertificate)> {
+fn read_cert(cert_path: &PathBuf) -> anyhow::Result<(String, Certificate)> {
     let mut cert_file = File::open(cert_path)?;
     let mut cert_content = String::new();
     cert_file.read_to_string(&mut cert_content)?;
     let cert_content = cert_content.trim();
-    let cert = golem_certificate::verify_golem_certificate(cert_content)?;
-    let id = str_to_short_hash(cert_content);
-    Ok((id, cert))
+    let cert = golem_certificate::validator::validate_golem_certificate(cert_content)?;
+    Ok((cert.chain[0].clone(), cert.cert))
```

Comment:
Not important, but in X509 chain cert_0 signs cert_1.
https://knowledge.digicert.com/solution/SO16297.html
_The chain or path begins with the SSL/TLS certificate, and each certificate in the chain is signed by the entity identified by the next certificate in the chain._

Reply:
I assume `chain[0]` is leaf certificate.